redisCache.py

```
@@ -106,4 +122,85 @@ async def redis_subscribe(redis: Redis, channel: str):
+    if not channel:
+        print("\033[91mERROR-DB:\033[0m Channel is empty.")
+        return None
+    if not message:
+        print("\033[91mERROR-DB:\033[0m Message is empty.")
+        return None
+
+    try:
+        pieces = [channel]
+        if maxlen is not None:
+            if maxlen < 0:
+                raise ValueError("maxlen must be a non-negative integer")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)